### ISSUE TITLE
Figure out a way to secure web sockets

### ISSUE THREAD

**Issue by wrathagom** [samtecspg/articulate#507]
Right now the websocket can be secured with SSL, but can't borrow the basic auth that we use for the UI. We need to figure out the easiest way to authenticate the client connection to the server.

May be related to #22, but ideally we wouldn't have to implement full on user management just to auth the socket.

**Comment by wrathagom** [comment]
@malave I am assuming this will be addressed with #598 ?

**Comment by malave** [comment]
@wrathagom The reverse proxy on #598 will just passthrough the request (http and ws) the security stuff will come with #600. Once the authentication schema is complete then we do the same (or similar) on the websocket.

**Comment by wrathagom** [comment]
Okay, closing this in favor of those issues.
